etc/scripts/install-on-minikube.sh

```diff
@@ -31,7 +31,19 @@ minikube config set cpus ${CPUS}
 minikube config set disk-size ${DISK_SIZE}
 
 # Start minikube
-minikube start -p knative --extra-config=apiserver.enable-admission-plugins="LimitRanger,NamespaceExists,NamespaceLifecycle,ResourceQuota,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook"
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
-"$DIR/install.sh" -q
+if [-z "$1"]; then
```

Comment:
I would prefer to stick to the convention of introducing a VM_DRIVER environment variable with a reasonable default that a user can override, similar to the existing MEMORY, CPUS, etc. That way, we wouldn't need the condition: we'd just always pass `--vm-driver ${VM_DRIVER}`. Make sense?

Reply:
Bonus points for setting the reasonable default based on the user's OS! :)

Reply:
Copying over excerpts from `minikube start --help`
```sh
      --vm-driver string                  VM driver is one of: [virtualbox parallels vmwarefusion kvm xhyve hyperv hyperkit kvm2 vmware none] (default "virtualbox")
```
I think we could just leave the default to "virtualbox" (as minikube does) irrespective of OS.

We check if `VM_DRIVER` env variable is available (and non-empty), and use it with `--vm-driver`.

WDYT?